kolibri/plugins/setup_wizard/assets/src/views/setup-forms/default-language-form/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+
+  <form @submit="setLanguage">
+    <fieldset class="default-language-form">
+      <legend>
+        <h1 class="default-language-form-header">
+          {{ $tr('languageFormHeader') }}
+        </h1>
+      </legend>
+
+      <label class="default-language-form-selected">
+        <span class="default-language-form-selected-label"> {{ $tr('selectedLanguageLabel') }} </span>
+        <span> {{ currentLanguage }} </span>
+      </label>
+
+      <k-button
+        v-for="language in buttonLanguages"
+        class="default-language-form-language-button"
+        :raised="false"
+        :text="language.name"/>
+
+      <label>
+        <span class="visuallyhidden">More Languages</span>
+        <select class="default-language-form-language-dropdown">
+          <option disabled selected> {{ $tr('showMoreLanguagesSelector') }} </option>
+          <option v-for="language in selectorLanguages" value="language.code">
+            {{ language.name }}
+          </option>
+        </select>
+      </label>
+
+      <k-button :primary="true" type="submit" :text="submitText" />
```

Comment:
Each of the forms is pretty similar aside from the fields, and I want to DRY it up. I'm debating between 2 approaches:

1. Having the common pieces in the `index` -- so `<form>` and `<k-button>` to minimize the props being passed around, then having each component be a `<fieldset>` at the top level. 
2. Creating a child component that includes the common bits and giving it a `<slot>` that each form will use

thoughts?

Reply:
Either sounds fine to me. (1) sounds simpler but a bit less flexible; probably what I would do for a first pass.